pkg/cmd/server/kubernetes/node_config.go

```diff
@@ -280,3 +288,30 @@ func BuildKubernetesNodeConfig(options configapi.NodeConfig) (*NodeConfig, error
 
 	return config, nil
 }
+
+// ensureDockerAPIVersionForParallelImagePulls returns true if the docker API version
+// is >=1.21, else false.
+func ensureDockerAPIVersionForParallelImagePulls(dh *docker.Helper, version string) bool {
```

Comment:
At this point, I dont have access to docker client, so I have to create it using docker helper.